docs/content/preview/develop/build-apps/java/ysql-vertx-pg-client.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Build a Java application that uses YSQL
+headerTitle: Build a Java application
+linkTitle: More examples
+description: Build a sample Java application with the Vertx PG Client and use the YSQL API to connect to and interact with YugabyteDB.
+menu:
+  preview:
+    parent: cloud-java
+    identifier: java-10
+    weight: 550
+type: docs
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li >
+    <a href="../ysql-yb-jdbc/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - YB - JDBC
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-jdbc/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - JDBC
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-vertx-pg-client/" class="nav-link active">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - Vertx PG Client
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-jdbc-ssl/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - JDBC SSL/TLS
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-hibernate/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - Hibernate
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-sdyb/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - Spring Data YugabyteDB
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-spring-data/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - Spring Data JPA
+    </a>
+  </li>
+  <li>
+    <a href="../ysql-ebean/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - Ebean
+    </a>
+  </li>
+  <li>
+    <a href="../ycql/" class="nav-link">
+      <i class="icon-cassandra" aria-hidden="true"></i>
+      YCQL
+    </a>
+  </li>
+  <li>
+    <a href="../ycql-4.6/" class="nav-link">
+      <i class="icon-cassandra" aria-hidden="true"></i>
+      YCQL (4.6)
+    </a>
+  </li>
+</ul>
+
+[Vertx PG Client](https://vertx.io/docs/vertx-pg-client/java/) is the client for PostgreSQL with simple APIs to communicate with the database. It is a reactive and non-blocking client for handling the database connections with a single threaded API.
+
+Because Vertx is a PostgreSQL client and YugabyteDB is PostgreSQL compatible, it is supported by YugabyteDB with a [limitation](#limitation).
```

Comment:
`Because Vertx is a PostgreSQL client and YugabyteDB is PostgreSQL compatible, it is supported by YugabyteDB with a [limitation](#limitation).`

Suggestion:
`Since YugabyteDB is wire compatible with PostgreSQL, Vertx PG Client works with YugabyteDB as well.`

And skip mentioning the limitation here itself since users will know about it as they go through rest of the page, IMO.

Reply:
Okay, that makes sense.